README.md

```diff
@@ -5,19 +5,21 @@ WSO2 EI Kafka inbound endpoint acts as a message consumer. It creates a connecti
 
 ## Compatibility
 
-| Inbound Endpoint version | Supported Kafka version | Supported WSO2 ESB/EI version |
-| ------------- | ---------------|------------- |
+| Inbound Endpoint version                                                                                 | Supported Kafka version | Supported WSO2 ESB/EI version |
+|----------------------------------------------------------------------------------------------------------| ---------------|------------- |
+| [1.0.12](https://github.com/wso2-extensions/esb-inbound-kafka/tree/org.apache.synapse.kafka.poll-1.0.10) | 2.12-2.2.1| EI 6.1.1, EI 6.4.0, EI 6.5.0  , MI 1.1.0  |
+| [1.0.11](https://github.com/wso2-extensions/esb-inbound-kafka/tree/org.apache.synapse.kafka.poll-1.0.10) | 2.12-2.2.1| EI 6.1.1, EI 6.4.0, EI 6.5.0  , MI 1.1.0  |
```

Comment:
branch version should be 1.0.11 and 1.0.12

Reply:
Updated